TODO
- actually check this patch against the disassembly.  The sha1sum did not
match the expected one, so something is wrong somewhere..

@@ -327714,7 +327714,7 @@
 00500210  00 00 00 00 00 00 00 00  00 00 00 00 00 00 00 00  |................|
 00500220  00 00 00 00 00 00 00 00  11 40 da 00 1c 67 63 0d  |.........@...gc.|
 00500230  00 00 00 00 00 00 00 00  00 00 00 00 00 00 00 00  |................|
-00500240  47 43 48 54 32 35 57 57  00 00 00 00 00 00 00 00  |GCHT25WW........|
+00500240  47 43 48 54 32 35 4d 43  00 00 00 00 00 00 00 00  |GCHT25MC........|
 00500250  03 16 05 30 01 00 15 00  00 c0 00 00 00 00 00 00  |...0............|
 00500260  00 00 00 00 00 00 00 00  28 43 29 20 43 6f 70 79  |........(C) Copy|
 00500270  72 69 67 68 74 20 49 42  4d 20 43 6f 72 70 2e 20  |right IBM Corp. |
@@ -335117,7 +335117,7 @@
 0051d0c0  71 17 00 f6 00 80 f8 1e  04 10 f8 16 01 11 42 29  |q.............B)|
 0051d0d0  00 02 d8 1e 02 10 d9 1e  42 10 1d ff 42 28 01 02  |........B...B(..|
 0051d0e0  da 1e 42 10 db 1e 02 10  c9 70 0a d9 3c da 04 db  |..B......p..<...|
-0051d0f0  18 f1 c4 16 01 10 c7 09  5e 80 c7 16 81 10 8c e9  |........^.......|
+0051d0f0  18 f1 c4 16 01 10 c7 09  5e 80 c7 16 81 10 e0 78  |........^......x|
 0051d100  88 e0 ca 21 81 0f 00 00  64 00 04 0f a1 f1 ca 20  |...!....d...... |
 0051d110  c1 03 0d f0 05 8e 01 e0  0f 78 88 e0 4e 02 2a 00  |.........x..N.*.|
 0051d120  05 ae 07 d8 c5 03 20 00  04 ae 7f 02 20 00 04 8e  |...... ..... ...|
@@ -335125,7 +335125,7 @@
 0051d140  43 1c 59 63 58 63 32 89  10 88 00 23 90 00 08 b9  |C.YcXc2....#....|
 0051d150  00 21 04 00 58 63 13 88  59 63 31 89 2f 24 08 01  |.!..Xc..Yc1./$..|
 0051d160  08 b8 38 60 10 78 88 71  ee 0f af f2 f8 16 02 11  |..8`.x.q........|
-0051d170  0b 08 51 00 00 86 83 b8  24 f0 10 10 80 20 11 10  |..Q.....$.... ..|
+0051d170  4a 26 00 70 00 86 83 b8  24 f0 10 10 80 20 11 10  |J&.p....$.... ..|
 0051d180  81 20 08 b8 38 60 12 10  81 20 10 78 13 10 82 20  |. ..8`... .x... |
 0051d190  08 b9 59 61 30 79 fa 0f  af f2 f8 16 02 11 0f 08  |..Ya0y..........|
 0051d1a0  51 00 00 86 45 20 00 0a  0e f0 05 8e 01 e0 0f 78  |Q...E .........x|
@@ -335141,12 +335141,12 @@
 0051d240  b0 d0 21 80 14 6a 58 60  14 79 4a 24 40 74 56 26  |..!..jX`.yJ$@tV&|
 0051d250  c2 16 a8 20 c0 01 01 11  80 04 01 1a 12 00 c9 70  |... ...........p|
 0051d260  0a d9 27 da bd 04 ef ff  11 db c4 16 00 10 e1 b8  |..'.............|
-0051d270  25 f3 c7 16 80 10 88 e8  e9 70 96 0d af f1 8a 21  |%........p.....!|
+0051d270  25 f3 c7 16 80 10 e0 78  e9 70 96 0d af f1 8a 21  |%......x.p.....!|
 0051d280  0f 0a 55 f1 05 8e 01 e0  0f 78 69 08 33 02 05 ae  |..U......xi.3...|
 0051d290  2e f0 da 0e af f1 e9 70  81 e0 11 f5 47 f1 c9 70  |.......p....G..p|
 0051d2a0  0b d9 7f 04 ef ff 28 da  08 16 c2 10 8c 22 ff 8f  |......(......"..|
 0051d2b0  04 f4 17 00 20 00 95 d1  92 d0 34 6a 59 61 3b 79  |.... .....4jYa;y|
-0051d2c0  01 80 11 e0 19 61 5e 0f  ef fe 56 26 c0 16 89 e8  |.....a^...V&....|
+0051d2c0  01 80 11 e0 19 61 5e 0f  ef fe 56 26 c0 16 e0 78  |.....a^...V&...x|
 0051d2d0  00 86 a4 b8 00 a6 14 d8  0f 02 20 00 04 ae 06 16  |.......... .....|
 0051d2e0  c0 10 61 b8 0d 78 0d 08  72 00 06 ae 0e d8 f6 f1  |..a..x..r.......|
 0051d2f0  00 86 84 b8 f0 f1 a5 ae  05 d8 04 ae c9 70 ac fd  |.............p..|
